### ISSUE TITLE
Internationalization:  Need to create key=value property file for Built-in Roles(JSON files)

### ISSUE THREAD

**Issue by JayanthyChengan** [IQSS/dataverse#5126]
Noticed that there are English terms in the built-in roles json files, which are loaded to database table(dataverserole) and displays in roles-assign.xhtml, roles-edit.xhtml and permission-manage.xhtml.  Planning to create a property files for Internationalization.

**Comment by scolapasta** [comment]
Hi,
I was looking for my review comment about Custom Roles, but couldn't find it. The issue here is that if you do have a custom role it would look for it in the bundle and not find it (rather than not display info from what is in db). Though if we remove the columns from the db, then that's a separate problem.

**Comment by JayanthyChengan** [comment]
commented here: https://github.com/IQSS/dataverse/pull/5158#discussion_r226311372

**Comment by pdurbin** [comment]
@JayanthyChengan I see @scolapasta was asking about custom roles. Are custom roles still supported? If so, what do installations need to do to continue using them and can you add something in the Admin Guide about custom roles? You can see the custom roles we use in Harvard Dataverse at https://dataverse.harvard.edu/role_permission_helper.xhtml

**Comment by JayanthyChengan** [comment]
Hi @pdurbin:  I am attaching the screenshot of permissions-manage.xhtml, where I concentrated for Roles/Permission Internationalization in this issue. 

In my test instance, custom roles are displayed under Roles tab in permission-manage.xhtml without any issue and there is no need for any additional installation steps.  

Created https://github.com/IQSS/dataverse/blob/28d05a43b68bf5be2a1abedf3ba935e22bc1216c/src/main/java/BuiltInRoles.properties -  the property file with roles name/description pulled from JSON files to handle multiple languages.

Also few keys related to permission were added to bundle.properties.

Thanks for pointing me to this page: role_permission_helper.xhtml, I was not aware of it( I will create a separate issue for this).  


![screen shot 2018-10-23 at 9 51 17 am](https://user-images.githubusercontent.com/12862735/47366249-50e90400-d6ab-11e8-875c-51fe4a5fdafe.png)
![screen shot 2018-10-23 at 9 51 26 am](https://user-images.githubusercontent.com/12862735/47366250-50e90400-d6ab-11e8-89cc-473d4919c475.png)
![screen shot 2018-10-23 at 9 52 03 am](https://user-images.githubusercontent.com/12862735/47366251-50e90400-d6ab-11e8-996a-f264814c6d89.png)
